metricflow/test/query/test_query_parser.py

```diff
@@ -249,6 +249,9 @@ def test_query_parser_case_insensitivity(bookings_query_parser: MetricFlowQueryP
         ),
     )
 
+    with pytest.raises(UnableToSatisfyQueryError):
+        bookings_query_parser.parse_and_validate_query(group_by_names=["random_stuff"])
```

Comment:
Why is this here? Should this be a separate test case?

Reply:
...laziness. I'll make a new test case LOL